schutzbot/mockbuild.sh

```diff
@@ -88,27 +88,27 @@ fi
 
 # Compile RPMs in a mock chroot
 greenprint "🎁 Building RPMs with mock"
-sudo mock -v -r $MOCK_CONFIG --resultdir $REPO_DIR --with=tests \
+sudo mock -v -r "$MOCK_CONFIG" --resultdir "$REPO_DIR" --with=tests \
     rpmbuild/SRPMS/*.src.rpm osbuild/rpmbuild/SRPMS/*.src.rpm
 
 # Change the ownership of all of our repo files from root to our CI user.
-sudo chown -R $USER ${REPO_DIR%%/*}
+sudo chown -R "$USER" "${REPO_DIR%%/*}"
 
 # Move the logs out of the way.
 greenprint "🧹 Retaining logs from mock build"
-mv ${REPO_DIR}/*.log $WORKSPACE
+mv "${REPO_DIR}"/*.log "$WORKSPACE"
 
 # Create a repo of the built RPMs.
 greenprint "⛓️ Creating dnf repository"
-createrepo_c ${REPO_DIR}
+createrepo_c "${REPO_DIR}"
 
 # Copy the current build to the latest directory.
-mkdir -p $REPO_DIR_LATEST
-cp -arv ${REPO_DIR}/ ${REPO_DIR_LATEST}/
+mkdir -p "$REPO_DIR_LATEST"
+cp -arv "${REPO_DIR}"/ "${REPO_DIR_LATEST}"/
 
 # Remove the previous latest build for this branch.
 # Don't fail if the path is missing.
-s3cmd --recursive rm s3://${REPO_BUCKET}/${JOB_NAME}/latest/${ID}${VERSION_ID//./}_${ARCH} || true
+s3cmd --recursive rm s3://${REPO_BUCKET}/"${JOB_NAME}"/latest/"${ID}""${VERSION_ID//./}"_"${ARCH}" || true
```

Comment:
```suggestion
s3cmd --recursive rm "s3://${REPO_BUCKET}/${JOB_NAME}/latest/${ID}${VERSION_ID//./}_${ARCH}" || true
```

Can we just quote the entire thing? Or is that not possible? FTR there are several cases of this in the PR.

Reply:
I think there should be no problem with it, but if the CI is happy with the current version I'd leave it as is and continue to improve our bash codebase in later PRs.

Anyway I changed at least this occurrence.